pkg/virt-config/virt-config.go

```diff
@@ -60,6 +60,7 @@ const (
 	SupportedGuestAgentVersions                     = "3.*,4.*"
 	DefaultOVMFPath                                 = "/usr/share/OVMF"
 	DefaultMemBalloonStatsPeriod                    = 10
+	DefaultCPUAllocationRatio                       = 10.0
```

Comment:
why 10 as the default?

Reply:
Well, just because right now 100m is the default in CNV templates and a number that recommended by the performance team[1]

https://bugzilla.redhat.com/show_bug.cgi?id=1679255#c12